test/integration/components.cc

```diff
@@ -966,7 +966,7 @@ TEST_F(ComponentsTest, LogicalMicrophone)
   comp1.Serialize(ostr);
   EXPECT_EQ("0 0.5", ostr.str());
 
-  std::istringstream istr;
+  std::istringstream istr(ostr.str());
```

Comment:
This change was made temporarily to get this test to pass - the real fix is in https://github.com/ignitionrobotics/ign-gazebo/pull/848. Once that PR is merged and forward-ported, I can revert this change.